lib/redis.js

```diff
@@ -3,27 +3,37 @@ const db = require('./connection')()
 
 class Redis {
 
-	constructor(table) {
-		this.TABLE = table;
+	get(key) {
+		return db.get(key);
 	}
 
-	find(key) {
-		return db.get(this.getKey(key));
-	}
-
-	insert(key, value) {
-		let k = this.getKey(key);
-		return db.set(k, value).then(() => {
-			return k;
-		})
+	/**
+	* Set query with expire flags
+	* @param {string} key - key for the redis entry
+	* @param {string} value - value for the redis entry
+	* @param {int} time - expire time in milisecondes if only 3 arguments
+	* @param {string} ex - if ex, expire time in secondes
+	* @returns {string} respond value from ioredis set function
+	*/
+	set(key, value, time, ex) {
```

Comment:
if the input is only time, then use miliseconds
if input is time and ex, i.e. set('key', 'value', 10, 's'), then use seconds, ex should also accept 'ms'